data/json/monster_special_attacks/spells.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "id": "pain_damage",
+    "type": "SPELL",
+    "name": "Pain",
+    "description": "Increases pain",
+    "valid_targets": [ "hostile" ],
+    "//": "Listed as a recover energy effect with a negative modifier that decreases with each level of the spell, which makes it cause damage instead.",
+    "min_damage": -50,
+    "max_damage": -50,
+    "damage_increment": -50,
```

Comment:
this spell has no levels, so the increment won't do anything. even if it did have levels, the max is the same as the min, making the increment also not do anything.

Reply:
So is the min/max thing not a random interval? This explains a lot.

Reply:
nope. min means "this is the number at level 0" then you add `increment` per level until you hit a maximum of `max`. (not necessarily at max level)

Reply:
Cool. We should update the docs about that probly... I could hit it while I do mapgen docs as part of this pr.

Reply:
my apologies if i didn't do a clear enough job in MAGIC.md

Reply:
no need, you developed it, it's hard to document something you wrote.